### PR TITLE
remove MSI

### PR DESCRIPTION
**Pull request by ejaz-alam**

### CODE REVIEW

File: help/inventory-management/install-update.md [hunk 1]
```diff
@@ -96,7 +96,62 @@ When complete, the system provides a list of modules and values in `<Magento_ins
    'Magento_InventoryAdvancedCheckout' => 0,
    ...
 ```
+## Remove Inventory Management
 
+Inventory Management moudule can be remove through composer file. To remove all MSI module simply add the following into composer.json
```
Comment (leslietilling) [inline help/inventory-management/install-update.md:101]:
```suggestion
Inventory Management module can be remove through composer file. To remove all Inventory Managements modules, simply add the following into composer.json.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
@ejaz-alam  Minor edit on the intro sentence. Can we also add a sentence about why someone would want to remove the modules? Are there any negative consequences to this?

**Comment by ejaz-alam** [comment]
> @ejaz-alam Minor edit on the intro sentence. Can we also add a sentence about why someone would want to remove the modules? Are there any negative consequences to this?

so sometimes we do not use the inventory modules and these modules are more than 20 modules which really affects the performance. so in terms performance, it could be uninstalled.

**Review by leslietilling — APPROVED** [review]
I am approving here and must then import to our internal repo, where I will make some minor grammatical and syntax edits before merging.

**Pull request merged**